include/swift/AST/DiagnosticEngine.h

```diff
@@ -309,8 +309,22 @@ namespace swift {
         : OpeningQuotationMark("'"), ClosingQuotationMark("'"),
           AKAFormatString("'%s' (aka '%s')"),
           OpaqueResultFormatString("'%s' (%s of '%s')") {}
+
+    /// When formatting fix-it arguments, don't include quotes or other
+    /// additions which would result in invalid code.
+    static DiagnosticFormatOptions formatForFixIts() {
+      return DiagnosticFormatOptions("", "", "%s", "%s");
+    }
   };
-  
+
+  enum class FixItID : uint32_t;
+
+  /// StructuredFixIt - Represents a fix-it defined in FixIts.def with a format
```

Comment:
Nitpick: please drop the type name from the doc comment. (I know we still have some old code that does it but we shouldn't propagate it.)